x/interchainstaking/client/cli/cli_test.go

```diff
@@ -66,6 +67,8 @@ func (s *IntegrationTestSuite) SetupSuite() {
 		MessagesPerTx:                0,
 		Is_118:                       true,
 	}
+
+	// TODO: I think setting validators here isn't enough, we need to set them in the store by using the keeper
```

Comment:
Correct; we used to keep them in the zone struct but it was horribly inefficient loading 500 vals every time we wanted to fetch the denom from a zone for example.